src/main/containers/MockupsPanel.jsx

```diff
@@ -96,6 +102,8 @@ const MockupsPanel = (props) => {
               >
                 {options}
               </Select>
+              <br />
+              <Button enabled onClick={saveFile}>Save Work as JSON</Button>
```

Comment:
Similar to opening a file, it’d be great to get this option in the file menu as well. I will also take a crack at some UI simplification but focusing on functionality for now 👍